plugins/modules/firewalld.py

```diff
@@ -748,7 +748,7 @@ def main():
             interface=dict(type='str'),
             masquerade=dict(type='str'),
             offline=dict(type='bool'),
-            target=dict(type='str', choices=['default', 'ACCEPT', 'DROP', 'REJECT']),
+            target=dict(type='str', choices=['default', 'ACCEPT', 'DROP', '%%REJECT%%']),
```

Comment:
Do you think we should keep the parameter as is and format value to %%REJECT%% when REJECT is provided by the user?

Reply:
On the background the %%REJECT%% is used, so I'd say use the %%REJECT%%. firewall-cmd seems to accept both REJECT and %%REJECT%% but in reality it sets it to %%REJECT%%

Reply:
if firewall-cmd accepts `REJECT` and `%%REJECT%%` both, can we add that as well?

Reply:
It accepts when running from terminal, when running through this module I got the 
```
Jun 29 09:49:39 <hostname retracted> ansible-ansible.posix.firewalld[9015]: Invoked with zone=public permanent=True state=enabled target=REJECT immediate=False timeout=0 icmp_[1850/7279] icmp_block_inversion=None service=None port=None port_forward=None rich_rule=None source=None interface=None masquerade=None offline=None                                                                 
Jun 29 09:49:39 <hostname retracted>  firewalld[915]: ERROR: INVALID_TARGET: REJECT
```
error, so I don't think so, and I am not sure what is difference between running it from cli and the module. 

Or do you want to automatically replace the REJECT with %%REJECT%%?

Reply:
Yes, if that is the case, we can replace REJECT with %%REJECT%% internally.

Reply:
I'd find that unwanted behavior. If I set something to REJECT and it doesn't return error I would assume the value is now REJECT and not something else...

If I then later come and check the value and compare it to REJECT but it is %%REJECT%% I would be confused... I'd only leave the %%REJECT%% which is what firewalld is anyways using to be crystal clear.

Reply:
OK.

Reply:
It's better to convert `REJECT` to `%%REJECT%%` and show a warning to user for now, remove convertion from `REJECT` to `%%REJECT%%` later in next release IMO.